modules/settings/settings.php

```diff
@@ -294,10 +298,67 @@ public function print_default_settings()
                 wp_nonce_field('edit-publishpress-settings');
 
                 submit_button(null, 'primary', 'submit', false); ?>
-            </form>
-            <?php
-
-            ?>
+                </form>
+                </div><!-- .pp-column-left -->
+                <?php if (!defined('PPCHPRO_VERSION')) { ?>
+                    <div class="pp-column-right">
+                        <div class="ppch-advertisement-right-sidebar">
```

Comment:
@olatechpro I have the same thought earlier, I checked on how Authors handle it, it was on Utils.php https://github.com/publishpress/PublishPress-Authors/blob/fd32088d5a0ff79bc82347c57de332240d5292e6/src/core/Classes/Utils.php#L1109

I have difficulty in separating it. I mean if I separate it, where should I put the function since we dont have Utils.php like Authors did.

Can you give me some hint or tips Ola?